### PR TITLE
Hacktoberfest - Fixup docker image name and tag

### PR DESCRIPTION
**Pull request by nwton**
As mentioned in https://www.jenkins.io/blog/2020/05/06/docker-agent-image-renaming/ and https://issues.jenkins-ci.org/browse/JENKINS-42846

Additional change old tag version from 3.35-5-alpine to 4.3-4-alpine (like commit 278798c02)

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]

File: examples/dood.groovy [hunk 1]
```diff
@@ -20,10 +20,10 @@ spec:
 """
   ) {
 
-  def image = "jenkins/jnlp-slave"
+  def image = "jenkins/inbound-agent"
   node(POD_LABEL) {
     stage('Build Docker image') {
-      git 'https://github.com/jenkinsci/docker-jnlp-slave.git'
+      git 'https://github.com/jenkinsci/docker-inbound-slave.git'
```
Comment (oleg-nenashev) [inline examples/dood.groovy:26]: docker-inbound-slave

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
One renaming is wrong, have not checked the full PR yet

File: examples/dood.groovy (hunk 1 above)
Comment (markyjackson-taulia) [reply examples/dood.groovy:26]: Good catch!

Comment (nwton) [reply examples/dood.groovy]: Sorry, this was wrong copy-paste, now I fix it.

**Review by oleg-nenashev — APPROVED** [review]
All changes look good to me. Thanks for the patch!

**Review by Vlatombe — APPROVED** [review]

**Comment by Vlatombe** [comment]
Thank you!

**Pull request merged**